test/functional/lib/tests.cpp

```diff
@@ -4348,6 +4349,7 @@ GenericRxFragmentBuffer(
     unique_fnmp_mtu_handle MtuReset;
     const UINT8 ThFlags = Params->TcpFlags != 0 ? Params->TcpFlags : TH_ACK;
     auto If = FnMpIf;
+    static const UINT8 TestNextHeaderValue = IPPROTO_GGP; // 0x03, obsolete.
```

Comment:
Either elaborate or remove this comment please. It's confusing as is.

Reply:
Found a better value and improved the comment.